### DISCUSSION TITLE
但求Lean大将核显驱动编译进去，硬件解码完全不生效TAT

### DISCUSSION THREAD

**Discussion by kiritoxjf** [coolsnowwolf/lede#9600] [R68S / 电犀牛专区]
/dev/dri/路径下存在card0和renderD128，但是配置了硬解就是不生效，所以怀疑是否是驱动问题TAT

**Comment by npcXxX** [comment]
同求

**Comment by kiritoxjf** [comment]
这是我目前查找到的可能有效的方法，RK家的CPU好像都是通用mpp库来实现硬件编解码，这是我找到的教程地址，可惜我看不懂这方面的开发，交给Lean大了  @coolsnowwolf [网址](https://t.rock-chips.com/forum.php?mod=viewthread&tid=336&extra=page%3D1)

**Reply by kiritoxjf** [reply]
[网址](http://arm-cn.com/thread-5236-1-1.html)
